gsutilwrap/__init__.py

```diff
@@ -38,7 +38,7 @@ def ls(pattern: str, dont_recurse: bool = False) -> List[str]:  # pylint: disabl
     if proc.returncode == 0:
         return [line.strip() for line in out.split('\n') if line.strip() != '']
 
-    raise RuntimeError("gsutil failed: command was: {}\n, stderr:\n{}".format(" ".join(
+    raise RuntimeError("gsutil failed: command was: {!r}\n, stderr:\n{}".format(" ".join(
```

Comment:
I found it sometimes frustrating that gsutilwrap (or subprocess or other modules) report the command as a list which I couldn't just copy/paste into terminal to verify. However, maybe you had a case where readability of the arguments (indention, special characters __etc.__) is more important than this copy/paste minor feature?

Reply:
I agree it is frustrating. My change was mainly motivated by being able to detect an empty command. I will keep your comment in mind and try to improve it when refractoring this module later this year